script/upgrades/MU01/tests/DeploymentChecks.sol

```diff
@@ -236,18 +231,19 @@ contract DeploymentChecks is Script, Test {
   }
 
   function verifyCircuitBreaker() public view {
```

Comment:
I took another look and actually think this logic needs to change, right now it checks whether every breaker is enabled for every rateFeed. In the previous deployment, we enabled the medianDeltaBreaker for the stable/celo rates and the ValueDeltaBreaker for cUSD/USDC. The previous verifyCircutBreaker was checking whether any breaker was set for each of the rateFeeds by checking the lastUpdatedTime on each rateFeed.

If we want to have the same logic with the new breaker box we would need to check `breakerBox.isBreakerEnabled(rateFeed[i], breakers[j])` or `breakerBox.rateFeedBreakerStatus(rateFeed[i], breakers[j]).enabled` and if this is not true for at least one of the breakers per rateFeed we  `revert("Not all breakers were set.");`. 

Also, I would suggest renaming the configuredBreakers to rate Feeds as they are not breakers.

Reply:
good catch! accidentally removed `isBreakerEnabled` check when refactoring. `configuredBreakers` were there before but will rename.